tests/data/Block.6.sample1.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Block.6.sample1.json, Block.7.sample1.json, Block.8.sample1.json
Why was it added?

Reply:
Since ballot transactions are being made dynamically, tx_hash get changed every time test runs, that's why in order to test ballot API's, these sample JSON files were added

Reply:
I would appreciate it if you could move the test to `Congress.test.ts`